generic/tests/netperf.py

```diff
@@ -185,6 +187,25 @@ def _pin_vm_threads(vm, node):
                     test.error(msg)
         if params.get("reboot_after_config", "yes") == "yes":
             session = vm.reboot(session=session, timeout=login_timeout)
+            vm.wait_for_serial_login(timeout=login_timeout, restart_network=True).close()
+
+    server_ip = vm.wait_for_get_address(0, timeout=90)
+    if len(params.get("nics", "").split()) > 1:
+        server_ctl = vm.wait_for_login(nic_index=1, timeout=login_timeout)
+        server_ctl_ip = vm.wait_for_get_address(1, timeout=90)
+    else:
+        server_ctl = vm.wait_for_login(timeout=login_timeout)
+        server_ctl_ip = server_ip
+
+    mac = vm.get_mac_address(0)
+    queues = int(params.get("queues", 1))
```

Comment:
I saw that the queues=4 is commented in the cfg file.
If the default value of queues is <= 4 of this case, ethtool -L will fail.
Only when default value is larger than 4, ethtool can set successfully.
I'd suggest that you take this into consideration. Add a comment to indicate what's the value available?

Reply:
Most of time, we tested with none-mq, so there will be no queues parma.
"# queues = 4" comment out in netperf.cfg